### PR TITLE
Feature/christophe/bencontrolflow

### PR DESCRIPTION
**Pull request by Christophe1965**
hi Ben, this is my work on control flow. Can you check it please?

### CODE REVIEW

File: BenControlFlow.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+//
+////1
+//var isItSnowing: Bool = true
+//
+//if isItSnowing {
+//
+//    print ("I can do snowboarding ")
+//} else {
+//print ("I can not do snwoboarding")
+//
```
Comment (benSmith1981) [inline BenControlFlow.playground/Contents.swift:14]: Indent your statements inside of the IF

File: BenControlFlow.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+//if doYouHaveTimeForBeginners {
+//   
+//    print (" I will be a good programmer")
+//} else {
+//    print (" I will have wasted my time")
+//}
+//
+//
+//var doYouHaveTimeForNewcomers: Bool = false
+//
```
Comment (benSmith1981) [inline BenControlFlow.playground/Contents.swift:38]: Yes I have time for beginners! :)

File: BenControlFlow.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+//if doYouHaveTimeForNewcomers {
+//   print (" I will be a good programmer")
```
Comment (benSmith1981) [inline BenControlFlow.playground/Contents.swift:40]: absolutely!

File: BenControlFlow.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+//    middleAged = false
+//}
+//
+////6
+//
+//var  middleAged1 = age >= 35 ? true: false
+//var  middleAged2 = age <= 35 ? true: false
+//var  middleAged3 = age == 35 ? true: false
+//var  middleAged4 = age != 35 ? true: false
+
```
Comment (benSmith1981) [inline BenControlFlow.playground/Contents.swift:121]: Well done great usage of tenary operators.... like to see you using different operators

File: Function Chapter 12 homework.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+     print("Hello \(name), welcome to your playground.")
+}
+printPersonalGreeting(to: "The Donald")
+
+func divisionDescriptionFor (numerator: Double, denominator: Double) {
+    print("\(numerator) divided by \(denominator) equals \(numerator / denominator)")
+    
+}
+divisionDescriptionFor(numerator: 9.0, denominator: 3.0)
+
```
Comment (benSmith1981) [inline Function Chapter 12 homework.playground/Contents.swift:36]: Good start with functions

**Review by benSmith1981 — APPROVED** [review]
Nice work! keep going, you can be a great programmer!

File: BenControlFlow.playground/Contents.swift (hunk 1 above)
Comment (Christophe1965) [reply BenControlFlow.playground/Contents.swift:14]: i forgot this then }

**Pull request merged**